### PR TITLE
fix: the registry certificate of the non master node was not updated …

### PR DESCRIPTION
**Pull request by sfeng1996**
### Does this pull request fix one issue?
https://github.com/sealerio/sealer/issues/1406

### CODE REVIEW

**Review by bxy4543 — APPROVED** [review]

File: apply/processor/gen.go [hunk 1]
```diff
@@ -187,7 +187,16 @@ func (g *GenerateProcessor) ApplyRegistry(cluster *v2.Cluster) error {
...
-	err = runt.(*runtime.KubeadmRuntime).GenerateRegistryCert()
+	rt, ok := runt.(*runtime.KubeadmRuntime)
+	if !ok {
+		return fmt.Errorf("valid runtime")
+	}
+	err = rt.GenerateRegistryCert()
+	if err != nil {
+		return err
+	}
+	// sealer need to update registry cert on other nodes, otherwise docker pull or push will report error
```
Comment (kakaZhou719) [inline apply/processor/gen.go:198]: here i think ,could we send to all nodes whether it is master0 or not after generate registry cert

Comment (fanux) [reply apply/processor/gen.go:198]: agress with you

Comment (sfeng1996) [reply apply/processor/gen.go:198]: ok，I'd like to ask how to realize it
send to master0  has been implemented in rt.GenerateRegistryCert()

Comment (kakaZhou719) [reply apply/processor/gen.go:198]: don't worry about this，how about we split it ? method "GenerateRegistryCert" only generate registry cert, and we use another method to do send .

**Pull request merged**